src/opt/modsecurity/activate-rules.sh

```diff
@@ -83,18 +81,4 @@ if [ ! -z $COMBINED_FILE_SIZES ]; then
   sed -z -E -i 's/#SecAction.{6}id:900350.*tx\.combined_file_sizes=1048576\"/SecAction \\\n  \"id:900350, \\\n   phase:1, \\\n   nolog, \\\n   pass, \\\n   t:none, \\\n   setvar:tx.combined_file_sizes='"$COMBINED_FILE_SIZES"'\"/' /etc/modsecurity.d/owasp-crs/crs-setup.conf
 fi 
 
-if [ $WEBSERVER = "Apache" ]; then
-  if [ ! -z $PROXY ]; then
-    if [ $PROXY -eq 1 ]; then
-      WEBSERVER_ARGUMENTS='-D crs_proxy'
-      if [ -z "$UPSTREAM" ]; then
-        export UPSTREAM=$(/sbin/ip route | grep ^default | perl -pe 's/^.*?via ([\d.]+).*/$1/g'):81
-      fi
-    fi
-  fi
-elif [ $WEBSERVER = "Nginx" ]; then
-  WEBSERVER_ARGUMENTS=''
-fi
-
-
-exec "$@" $WEBSERVER_ARGUMENTS
+exec "$@"
```

Comment:
This script doesn't need to call a command passed in at all. This line, which originally comes from the entrypoint script, should thus be removed.